cluster/gce/config-test.sh

```diff
@@ -360,6 +360,27 @@ if [[ ! -z "${NODE_ACCELERATORS}" ]]; then
     fi
 fi
 
+# Arktos specific network and service support is a feature that each
+ # pod is associated to certain network, which has its own DNS service.
+ # By default, this feature is enabled in the dev cluster started by this script.
+DISABLE_NETWORK_SERVICE_SUPPORT="${DISABLE_NETWORK_SERVICE_SUPPORT:-}"
+DISABLE_ADMISSION_PLUGINS=${DISABLE_ADMISSION_PLUGINS:-""}
+ARKTOS_NETWORK_TEMPLATE="${KUBE_ROOT}/hack/testdata/default-flat-network.tmpl"
```

Comment:
if I understand correct, the var "ARKTOS_NETWORK_TEMPLATE" only used when DISABLE_NETWORK_SERVICE_SUPPORT != true. for default value, suggest to keep null.

Reply:
We make the change as we discuss. Thanks!